autoload/db_ui.vim

```diff
@@ -103,7 +103,7 @@ function! db_ui#query(query) abort
     throw 'Unsupported scheme '.parsed.scheme
   endif
 
-  let result = db_ui#schemas#query({ 'conn': b:db }, printf(scheme.args, a:query))
+  let result = db_ui#schemas#query({ 'conn': b:db, 'scheme': parsed.scheme }, printf(scheme.args, a:query))
```

Comment:
Whenever running a custom `#query()`, the 'scheme' field should be provided now. This is because of `sqlplus`'s requirement of piping queries to it, rather than using a flag. As such, the `#query()` function needs to know if queries come before or after the sql command, so the 'scheme' must be present.

I have adjusted all instances of this within the project.